Updating dropdown topic (#417)
IgniteUI/indigo-design-docfx: en/components/dropdown.md

# Dropdown

> Use the Dropdown Component to let the user select an item from a collection that is displayed upon user interaction in a scrollable list. Only one item can be selected at a time and if your scenario allows for multiple selections, you should use the [Combo](combo.md). The Dropdown is visually identical to the [Ignite UI for Angular Dropdown Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop_down.html)

Reviewer: ```suggestion
Use the Dropdown Component to let the user select an item from a collection that is displayed upon user interaction in a scrollable list. Only one item can be selected at a time and if your scenario requires the selection of multiple items, you should use the [Combo](combo.md) instead. The Dropdown is visually identical to the [Ignite UI for Angular Dropdown Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop_down.html)
```

## Item States

The Dropdown items support five different states: disabled, **idle**, focused, selected and selected&focused.

<img class="responsive-img" src="../images/dropdown_item_disabled.png" srcset="../images/dropdown_item_disabled@2x.png 2x" />
<img class="responsive-img" src="../images/dropdown_item_idle.png" srcset="../images/dropdown_item_idle@2x.png 2x" />
<img class="responsive-img" src="../images/dropdown_item_focused.png" srcset="../images/dropdown_item_focused@2x.png 2x" />
<img class="responsive-img" src="../images/dropdown_item_selected.png" srcset="../images/dropdown_item_selected@2x.png 2x" />
<img class="responsive-img" src="../images/dropdown_item_selected_focused.png" srcset="../images/dropdown_item_selected_focused@2x.png 2x" />

## Item Layout Template

The Dropdown Item supports flexible icon and label templating achievable by setting the elements you want to hide to ~No Symbol from the overrides panel. By default the icon is hidden with this approach.

> <img class="responsive-img" src="../images/dropdown_item_idle.png" srcset="../images/dropdown_item_idle@2x.png 2x" /> 

Reviewer: add a second image with both icon and label

Reviewer: only adding a second image here seems to be left and I will approve the changes

## Item States

> The Dropdown items support five different states: disabled, **idle**, focused, selected and selected&focused.

Reviewer: ```suggestion
The Dropdown items support five different states: Disabled, **Idle**, Focused, Selected and Selected&focused.
```

## Usage

> When using the Dropdown you should always show it on top of the content that follows the item which triggers its display. Dropdowns don't push content like accordions do.

Reviewer: ```suggestion
When using the Dropdown you should always show it on top of the content that follows the item which triggers its display. Dropdowns don't push content like an expansion panel would.
```
